conbench/templates/app.html

```diff
@@ -1,17 +1,22 @@
-{% extends 'bootstrap/base.html' %}
-
-{% block title %}
-    {% if title %}{{ title }} - {{ application }}{% else %}{{ application }}{% endif %}
-{% endblock %}
-
+{% block doc -%}
+<!doctype html>
+<html lang="en">
+{%- block html %}
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
 {% block styles %}
-{{super()}}
 <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}">
 <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
 <link href="https://cdn.datatables.net/1.10.23/css/dataTables.bootstrap.min.css" rel="stylesheet" type="text/css">
 <link href="https://cdn.datatables.net/responsive/2.2.7/css/responsive.bootstrap.min.css" rel="stylesheet" type="text/css">
 <link href="{{ url_for('static', filename='app.css') }}" rel=stylesheet type=text/css />
 {% endblock %}
+<title>{% block title %}{% if title %}{{ application }} - {{ title }}{% else %}{{ application }}{% endif %}{% endblock %}</title>
```

Comment:
I sneaked a title construction change into this PR. Note that this title defines the browser tab label.

In the title I like having the application name (e.g. `Conbench Staging`) to be before something like `Home`. So that when I have many tabs open in my browser, I find discovery easier.